authentication/views.py

```diff
@@ -1,13 +1,81 @@
-from django.shortcuts import render
 from django.contrib.auth import login,logout,authenticate
+from django.shortcuts import render,redirect
+from django.contrib import messages 
+from django.contrib.auth.models import User,auth 
 # Create your views here.
 
 
-def loginView(request):
-    pass
+def loginUser(request):
+    
+  class given :
+     
+      def __init__(self,u,p):
+        self.u = u
+        self.p = p
 
-def logoutView(request):
-    pass
+  if(request.method == 'POST'):
+       username=request.POST['username'] 
+       password=request.POST['password']
```

Comment:
You are directly accessing POST data without checking if it even exists. This may lead to server crash if a user access this endpoint with invalid request data. The good behavior would have been to throw a client error (400), rather than server error (500).